imos_bgc_db/NRS_StationInfo.resource.yaml

```diff
@@ -0,0 +1,52 @@
+encoding: utf-8
```

Comment:
I've only included this file as an example to show that we can specify more than just the schema, which _may_ be useful at some point (see https://specs.frictionlessdata.io/tabular-data-resource). Note that this now refers to a specific file at the location specified by the `path`.